### ISSUE TITLE
multi-channel images

### ISSUE THREAD

**Issue by elnazazmi** [holmdk/Video-Prediction-using-PyTorch#7]
Hi,

I would like to ask about using this model with a data set of multi-channel images. Which parts of the code should be changed to adjust this code for multi-channel images?

**Comment by holmdk** [comment]
Hi. 

Go into this [file](https://github.com/holmdk/Video-Prediction-using-PyTorch/blob/master/models/seq2seq_ConvLSTM.py)

`self.encoder_1_convlstm = ConvLSTMCell(input_dim=in_chan,
                                       hidden_dim=nf,
                                       kernel_size=(3, 3),
                                       bias=True)`

Set in_chan=3


And also in this code block:

`self.decoder_CNN = nn.Conv3d(in_channels=nf,
                                     out_channels=1,
                                     kernel_size=(1, 3, 3),
                                     padding=(0, 1, 1))`
Set out_channels=3


I'm pretty sure that is it!
